nix/cells/automation/pipelines.nix

```diff
@@ -0,0 +1,73 @@
+{ cell
+, inputs
+}:
+let
+  inherit (inputs.nixpkgs) system;
+
+  common =
+    { config
+    , ...
+    }: {
+      preset = {
+        nix.enable = true;
+
+        github-ci = {
+          # Tullia tasks can run locally or on Cicero.
+          # When no facts are present we know that we are running locally and vice versa.
+          # When running locally, the current directory is already bind-mounted
+          # into the container, so we don't need to fetch the source from GitHub
+          # and we don't want to report a GitHub status.
+          enable = config.actionRun.facts != { };
+          repo = "input-output-hk/plutus";
+        };
+      };
+
+      memory = 1024 * 8;
+      nomad.resources.cpu = 10000;
+    };
+in
+{
+  ci = { config, lib, ... }: {
+    imports = [ common ];
+
+    preset.github-ci.sha =
+      config.preset.github-ci.lib.readRevision inputs.cells.cloud.library.actions.ci.input null;
+
+    command.text = ''
+      nix build -L .#${lib.escapeShellArg system}.automation.ciJobs.required
+    '';
+  };
+
+  publish-unraveling-recursion-paper = { config, pkgs, lib, ... }: {
+    imports = [ common ];
```

Comment:
This is potentially going to provision a massively OTT machine for just publishing an artifact, unsure if that's a problem or those resource requirements are real or what.

Reply:
Right, the publish task should need much less because it can just pull the artifact from the cache.